test/src/test/kotlin/graphql/nadel/tests/EngineTests.kt

```diff
@@ -64,7 +64,11 @@ class EngineTests : FunSpec({
                     // Rename fixtures if they have the wrong name
                     val expectedName = fixture.name.toSlug()
                     if (file.nameWithoutExtension != expectedName) {
-                        file.renameTo(File(file.parentFile, "$expectedName.${file.extension}"))
+                        val expectedFile = File(file.parentFile, "$expectedName.${file.extension}")
```

Comment:
is that why sometimes my test files were being deleted? 🤔

Reply:
Ahh yes very sorry. It was supposed to be a cleanup thing so we could keep names consistent but I didn't think about what would happen when you copy pasted files.

I had a few tests disappear on me too :(